clippy_lints/src/missing_const_for_fn.rs

```diff
@@ -101,6 +101,11 @@ impl<'tcx> LateLintPass<'tcx> for MissingConstForFn {
             return;
         }
 
+        let hir_id = cx.tcx.local_def_id_to_hir_id(def_id);
+        if is_in_test(cx.tcx, hir_id) {
```

Comment:
nit: I don't think this is a cheap check, we should have this later on, before the `optimized_mir` call.

Reply:
Moved